tests/input.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { screen, render, cleanup} from "@testing-library/react";
+import AppTesting from "../src/components/input";
```

Comment:
Why would you name this component `input` since it's not an input? naming conversion should be implied.
A component should have a name which really describes what it's.

Reply:
Thank you. I was trying to create an extra component to be used while checking if the testing environment I have configured is working.